src/credentials/awsCredentialsStatusBarItem.ts

```diff
@@ -29,25 +29,33 @@ export async function initializeAwsCredentialsStatusBarItem(
         'The current credentials used by the {0} Toolkit.\n\nClick this status bar item to use different credentials.',
         getIdeProperties().company
     )
+
     statusBarItem.show()
     updateCredentialsStatusBarItem(statusBarItem)
 
     context.subscriptions.push(statusBarItem)
 
     context.subscriptions.push(
-        awsContext.onDidChangeContext(async (awsContextChangedEvent: ContextChangeEventsArgs) => {
-            updateCredentialsStatusBarItem(statusBarItem, awsContextChangedEvent.profileName)
+        awsContext.onDidChangeContext(async (ev: ContextChangeEventsArgs) => {
+            updateCredentialsStatusBarItem(statusBarItem, ev.profileName, ev.developerMode)
         })
     )
 }
 
 // Resolves when the status bar reaches its final state
 export async function updateCredentialsStatusBarItem(
     statusBarItem: vscode.StatusBarItem,
```

Comment:
```suggestion
    statusBarItem: vscode.StatusBarItem & { backgroundColor: vscode.StatusBarItem['color'] },
```

Reply:
that would require `as` earlier in the chain. Which spreads the issue.
This will go away when we update. Prefer to keep it isolated here.

Reply:
Ah right you'd have to mark it as `backgroundColor?:`
But yeah not a big deal.